server/jetstream_cluster.go

```diff
@@ -4445,7 +4445,7 @@ func (js *jetStream) processClusterDeleteConsumer(ca *consumerAssignment, isMemb
 	if acc, _ = s.LookupAccount(ca.Client.serviceAccount()); acc != nil {
 		if mset, _ := acc.lookupStream(ca.Stream); mset != nil {
 			if o := mset.lookupConsumer(ca.Name); o != nil {
-				err = o.stopWithFlags(true, false, true, wasLeader)
+				err = o.stopWithFlags(true, false, true, wasLeader, false)
```

Comment:
If we can dynamically discover this spinning off of the delete flag would prefer that instead of adding new bool to this function call.